Automattic/wp-calypso
# Gutenberg: Make MoreMenu as similar as possible between Calypso and WPCOM

The Core `MoreMenu` lives inside the `@wordpress/edit-post` package, but it's not designed to be reused (and in fact it's not even exported).
Since there are several differences between the Core and Calypsoify implementations (e.g. in Calypsoify there are no fullscreen and manage reusable blocks options), I've decided to recreate it in Calypso.

Currently, the items "Keyboard Shortcuts" doesn't work, but we'll take care of it later.

#### Changes proposed in this Pull Request

* Add roughly the same `MoreMenu` components from Core into Calypso.
* Move them all outside of `edit-post`.

#### Testing instructions



* Open the native Gutenberg in Calypso integration: http://calypso.localhost:3000/gutenberg/post
* In a different tab, open Gutenberg on a WPCOM site: `/wp-admin/post-new.php`
* Make sure the two `MoreMenu` (the vertical ellipsis menu in the top-right corner) implementations look and behave the same. 

Fixes #28726
